com.freerdp.FreeRDP.json

```diff
@@ -0,0 +1,177 @@
+            "--enable-shared",
+            "--with-systemdsystemunitdir=/app/lib/systemd/"
+        ],
+        "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/LudovicRousseau/PCSC.git",
+                "tag": "1.9.9",
+                "commit": "15c16c7796607b1c8a2ce253d3f536918ab26b4a"
+            }
+        ],
+				"cleanup": [
+					"/share/doc",
+					"/share/man"
+				]
+    }, 
+		{
+			"name": "openssl111",
+			"buildsystem": "simple", 
+			"build-commands": [
+				"./config --prefix=/app",
+				"make -j $FLATPAK_BUILDER_N_JOBS",
+				"make install"
+			],
+			"cleanup": [
+				"/share/doc",
+				"/share/man",
+				"/bin"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.openssl.org/source/openssl-1.1.1q.tar.gz",
+					"sha256": "d7939ce614029cdff0b6c20f0e2e5703158a489a72b2507b8bd51bf8c8fd10ca"
+				}
+			]
+		},
+    {
+        "name": "freerdp",
+        "buildsystem": "cmake-ninja",
+        "cleanup": [
+        ],
+        "config-opts": [
+					"-DCMAKE_VERBOSE_MAKEFILE=ON", 
+					"-DCMAKE_BUILD_TYPE:STRING=Release",
+					"-DCMAKE_INSTALL_LIBDIR:PATH=lib",
+					"-DCHANNEL_TSMF:BOOL=OFF",
+					"-DCHANNEL_URBDRC:BOOL=ON",
+					"-DBUILD_TESTING:BOOL=OFF",
+					"-DWITH_MANPAGES:BOOL=OFF",
+					"-DWITH_GSSAPI:BOOL=OFF",
+					"-DWITH_PCSC:BOOL=ON",
+					"-DWITH_PKCS11:BOOL=ON",
+					"-DWITH_SWSCALE:BOOL=ON",
+					"-DWITH_SERVER:BOOL=ON",
+					"-DWITH_SAMPLE:BOOL=ON",
+					"-DWITH_CUPS:BOOL=ON",
+					"-DWITH_FFMPEG:BOOL=ON",
+					"-DWITH_DSP_FFMPEG:BOOL=ON",
+					"-DWITH_OSS:BOOL=OFF",
+					"-DWITH_PULSE:BOOL=ON"
+        ],
+        "sources": [
+        {
+            "type": "git",
+            "url": "https://github.com/FreeRDP/FreeRDP.git",
+            "tag": "2.8.0",
+            "commit": "e3fc97feb512053189e276b2ca79762990bb8c4c"
+        }
+        ]
+    },
+    {
+        "name": "freerdp-icons",
+        "buildsystem": "simple",
+        "cleanup": [
+        ],
+         "build-commands": [
+                "install -D resources/FreeRDP_Logo_Icon.svg /app/share/icons/hicolor/scalable/apps/FreeRDP_Logo_Icon.svg",
+                "install -D resources/FreeRDP_Logo.svg /app/bin/FreeRDP_Logo.svg",
+                "install -D resources/FreeRDP_Icon.svg /app/bin/FreeRDP_Icon.svg"
+        ],
+        "sources": [
+        {
+            "type": "git",
+            "url": "https://github.com/FreeRDP/FreeRDP.git",
+            "tag": "2.8.0",
+            "commit": "e3fc97feb512053189e276b2ca79762990bb8c4c"
+        }
+        ]
+    },
+
+		{
+        "name": "wrapper",
+        "buildsystem": "simple",
+        "build-commands": [
+                "install -D freerdp.sh /app/bin/freerdp.sh",
+                "install -D com.freerdp.FreeRDP.appdata.xml /app/share/appdata/com.freerdp.FreeRDP.appdata.xml",
+                "install -D com.freerdp.FreeRDP.desktop /app/share/applications/com.freerdp.FreeRDP.desktop",
```

Comment:
it surely won't find these two files.

also this should be part of the main module too.

Reply:
that's the build failure btw.